lib/Onnxifi/Base.cpp

```diff
@@ -17,13 +17,52 @@
 
 #include "glow/Importer/ONNXIFIModelLoader.h"
 
+#include "llvm/ADT/ScopeExit.h"
 #include "llvm/ADT/SmallVector.h"
 
 namespace glow {
 namespace onnxifi {
+onnxStatus BackendId::checkGraphCompatibility(const void *onnxModel,
```

Comment:
Do we compile or we just do verification?

Reply:
We only import onnx graph to glow graph then lower the glow graph, no optimization, no compiling of any kind. I'll update the comment in Base.h to say this.